{"uid":"c2fbae410fa39f31","name":"Deve realizar o upload do arquivo 90.jpg","fullName":"\"before all\" hook for \"Deve realizar o upload do arquivo 90.jpg\"","historyId":"d79a85090473098ac489e22724d10a66","time":{"start":1755874778985,"stop":1755874778988,"duration":3},"status":"broken","statusMessage":"The following error originated from your application code, not from Cypress.\n\n  > angular is not defined\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"ReferenceError: The following error originated from your application code, not from Cypress.\n\n  > angular is not defined\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at <unknown> (https://demo.automationtesting.in/angcontroller.js:6:9)","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"time":{},"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress.\n\n  > angular is not defined\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"ReferenceError: The following error originated from your application code, not from Cypress.\n\n  > angular is not defined\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at <unknown> (https://demo.automationtesting.in/angcontroller.js:6:9)","steps":[{"name":"visit (\"https://demo.automationtesting.in\")","time":{"start":1755874776956},"status":"unknown","steps":[],"attachments":[],"parameters":[],"attachmentStep":false,"stepsCount":0,"hasContent":false,"attachmentsCount":0,"shouldDisplayMessage":false}],"attachments":[],"parameters":[],"attachmentStep":false,"stepsCount":1,"hasContent":true,"attachmentsCount":0,"shouldDisplayMessage":true}],"afterStages":[],"labels":[{"name":"package","value":"cypress.integration.file_upload_test.spec.js"},{"name":"suite","value":"CY0019 File Upload Test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[]}],"tags":[]},"source":"c2fbae410fa39f31.json","parameterValues":[]}